rpc_jobs/rpc_aio.yml

```diff
@@ -284,8 +285,19 @@
             if (env.SERIES == "kilo"
                 && env.TRIGGER == "pr"
                 && env.ACTION == "leapfrogupgrade" ){{
+                kibana_branch = env.UPGRADE_FROM_REF
                 env.UPGRADE_FROM_REF="origin/pr/${{env.ghprbPullId}}/merge"
             }}
+            // Here we set kibana_branch when it has not been set above
+            if (kibana_branch == null) {{
+              if (env.STAGES.contains("Minor Upgrade")
+                  || env.STAGES.contains("Major Upgrade")
+                  || env.STAGES.contains("Leapfrog Upgrade")) {{
+                kibana_branch = env.UPGRADE_FROM_REF
```

Comment:
If we test kibana after the leapfrog, should we need this, isn't this contrary to the expectations?